### DISCUSSION TITLE
Pushbullet params

### DISCUSSION THREAD

**Discussion by Adelscott** [containrrr/shoutrrr#176] [Q&A] [answered]
Hi,
Is there any  params we can use with Pusbhullet notifications ?

I'm trying to set the type, title, body etc ... but it's not working.  Are they not [implemented](https://containrrr.dev/shoutrrr/services/pushbullet/) when the title seems to indicate otherwise?

**Comment by piksel** [comment, marked as answer]
Do you mean the "Query/Param Props" title? It's a generic text that precedes all valid properties that can be set, but in the case of pushbullet there are none.
It looks like title is half-implemented, but I'm not sure why. Shouldn't be too much effort to add support though.

**Reply by Adelscott** [reply]
Yes, that title ! That would be a nice feature, but no need to rush, I can live without it !
